src/ontology/mondo-edit.obo

```diff
@@ -421772,8 +421766,10 @@ xref: NANDO:1200583 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0
 xref: NANDO:2201295 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0011-764X", source="https://orcid.org/0000-0002-0170-9172"}
 xref: Orphanet:447893 {source="MONDO:equivalentTo"}
 xref: UMLS:C5681201 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1842862"}
+is_a: MONDO:0011897 {source="https://orcid.org/0000-0001-5493-2602"} ! leukodystrophy, hypomyelinating, 7, with or without oligodontia and/or hypogonadotropic hypogonadism
 is_a: MONDO:0021147 {source="MONDO:0017122-obsoleted"} ! disorder of development or morphogenesis
-relationship: excluded_subClassOf MONDO:0011897 {source="https://orcid.org/0000-0001-5208-3432"} ! leukoencephalopathy-ataxia-hypodontia-hypomyelination syndrome
+relationship: excluded_subClassOf MONDO:0011897 {source="https://orcid.org/0000-0001-5208-3432"} ! leukodystrophy, hypomyelinating, 7, with or without oligodontia and/or hypogonadotropic hypogonadism
```

Comment:
This SC is both an actual subclass and excluded. There is only an orcid as source for both this SC and excluded SC. 
This should be reviewed and 1) updated, 2) a more specific source (if possible a PMID or a reference other than an orcid) should be added.

Reply:
good catch!